app/src/main/java/com/farmerbb/taskbar/util/U.java

```diff
@@ -1111,8 +1111,20 @@ public static boolean isBlissOs(Context context) {
                 && isSystemApp(context);
     }
 
+    public static boolean isProjectSakura(Context context) {
+        boolean validSakuraBuildProp = false;
+
+        String sakuraVersion = getSystemProperty("ro.lineage.build.version");
+        if(sakuraVersion != null && !sakuraVersion.isEmpty())
```

Comment:
What about adding { and } for if?

Reply:
Can you help me algining it ?

Reply:
I think it's better you changed it if you think it is deserved. My review suggestion is only suggestion. You can omit it, if @farmerbb  think current changes are fine.

Reply:
@hmtheboy154 I can fix the brace issues once this is merged